projects/museumpassmusees/reference/partner-api.json

```diff
@@ -430,9 +430,188 @@
           "Visits"
         ]
       }
+    },
+    "/external-cards/{type}/{identification}": {
+      "get": {
+        "summary": "Look up external card",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "visit": {
+                      "type": "object",
+                      "required": [
+                        "allowed"
+                      ],
+                      "description": "Visit information for the external card.",
+                      "properties": {
+                        "allowed": {
+                          "type": "boolean",
+                          "description": "Indicates whether a visit with the external card is allowed or not."
+                        },
+                        "reason": {
+                          "type": "string",
+                          "enum": [
+                            "NO_SOCIAL_STATUS"
```

Comment:
Are `EXPIRED` and `BLOCKED` not possible reasons in the case of UiTPAS and/or A-Kaart?

Reply:
From a functional perspective MPM only has to check for 'social status' - the question is: if we would need to extend this later, is it better to include EXPIRED and BLOCKED now (but ignore them in the implementation) or add them when needed. (extra enum values could be seen as a breaking change)

Reply:
As discussed on Slack the `reason` is just left out for now, since it doesn't give any extra info if it can only be `NO_SOCIAL_STATUS` for now.